Dockerfile

```diff
@@ -49,5 +52,8 @@ COPY --from=0 /server/package.json server/package.json
 COPY --from=0 /server/build server/build
 COPY --from=0 /client/build client/build
 COPY --from=0 /client/index.html client/index.html
+COPY --from=0 /etc/passwd /etc/passwd
```

Comment:
@alex1989hu thanks for PR!  One question:
is this just to propagate useradd from a build container?

Reply:
Yes, that's all. It is easier to copy a valid file, than `RUN echo ...` => much more less possible typo or syntax error